src/backend/lima.ts

```diff
@@ -2154,8 +2191,35 @@ class LimaKubernetesBackend extends events.EventEmitter implements K8s.Kubernete
     await this.vm.writeFile('/etc/logrotate.d/k3s', LOGROTATE_K3S_SCRIPT);
   }
 
-  protected async deleteIncompatibleData(isDowngrade: boolean) {
-    if (isDowngrade) {
+  /**
+   * Persist the given version into the VM disk, so we can look it up later.
+   */
+  protected async persistVersion(version: semver.SemVer): Promise<void> {
+    const filepath = '/var/lib/rancher/k3s/version';
```

Comment:
Should this be a global constant, so it doesn't have to be defined multiple times?

Reply:
Switched to `k3s --version`, so this is removed.